ui/store/actions.ts

```diff
@@ -3119,7 +3119,7 @@ export function setShowTokenAutodetectModal(value: boolean) {
   return setPreference('showTokenAutodetectModal', value);
 }
 
-export function setAutoLockTimeLimit(value: boolean) {
+export function setAutoLockTimeLimit(value: number) {
```

Comment:
It looks like this could be invoked with `null` as well

```suggestion
export function setAutoLockTimeLimit(value: number | null) {
```